# Fix compatibilty with older libstdc++ libraries, such as 6.3.0 in Debian stretch

--- a/device/fido/fido_hid_device.cc
+++ b/device/fido/fido_hid_device.cc
@@ -40,7 +40,7 @@ void FidoHidDevice::Cancel() {
     return;
 
   // Delete any remaining pending requests on this Channel ID.
-  pending_transactions_ = {};
+  pending_transactions_ = base::queue<std::pair<std::vector<uint8_t>, DeviceCallback>>();
   WriteMessage(
       FidoHidMessage::Create(channel_id_, FidoHidDeviceCommand::kCancel,
                              std::vector<uint8_t>()),
